site/docs/v1/tech/apis/scim/SCIMEnterpriseUser.adoc

```diff
@@ -0,0 +1,159 @@
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
+
+:create_user:
+include::/docs/v1/tech/apis/scim/_scim-enterpriseuser-response-body.adoc[]
+:create_user!:
+
+== Delete a EnterpriseUser
+
+This API is used to delete a FusionAuth User through a SCIM request. You must specify the Id of the User on the URI.
+
+The data of a User who has been hard deleted is permanently removed from FusionAuth. The User's data cannot be restored via the FusionAuth API or the administrative user interface. If you need to restore the User's data, you must retrieve it from a database backup.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Delete a EnterpriseUser through a SCIM request
+[.endpoint]
+.URI
+--
+[method]#DELETE# [uri]#/api/scim/resources/v2/EnterpriseUsers`\{userId\}`#
+--
+==== Request Parameters
+
+[.api]
+[field]#userId# [type]#[UUID]# [optional]#Optional#::
+The FusionAuth userId.
+
+:http_method: DELETE
+
+=== Response
+
+This API does not return a JSON response body.
+
+The DELETE endpoint will return a 204 status code upon success or one of the standard error status codes.
+
+:webhook_event:
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
+
+== Retrieve a EnterpriseUser
+
+This API is used to retrieve a FusionAuth User in SCIM schema format through a SCIM request.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieves a EnterpriseUser through a SCIM request
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/scim/resources/v2/EnterpriseUsers/`\{userId\}`#
+--
+==== Request Parameters
+
+[.api]
+[field]#userId# [type]#[UUID]# [optional]#Optional#::
+The FusionAuth userId.
+
+:http_method: GET
+
+=== Response
+
+The response for this API contains the User in SCIM schema format.
+
+:webhook_event:
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
+
+:create_user:
+include::/docs/v1/tech/apis/scim/_scim-enterpriseuser-response-body.adoc[]
+:create_user!:
+
+== Update a EnterpriseUser
+
+This API is used to update a new FusionAuth User from a SCIM request. This is a true update meaning that the FusionAuth User will be overwritten with only the data contained in the request. It is NOT a partial update.
```

Comment:
```suggestion
This API is used to update a new FusionAuth User from a SCIM request. The FusionAuth User will be overwritten by  the data contained in the request. It is not a partial update or a patch. The only exceptions are ...
```

I assume we aren't updating the password, salt, encryptionScheme fields with this? Or are we?

Reply:
It will actually update the password per SCIM specification